mysql-test/t/global_read_lock.inc

```diff
@@ -1,8 +1,14 @@
+--echo # Test setting read only with long running transactions engine=$engine,
```

Comment:
Does this need blackhole_transactions=ON?

Reply:
Yes, perhaps it is OK as it does technically have transaction support but not enabled. However, I can also see the other side where it can be confusing.